fusillade/clouddirectory.py

```diff
@@ -1320,11 +1320,13 @@ def get_authz_params(self) -> Dict[str, Union[List[Dict[str, str]], List[str]]]:
         policy_paths = self.cd.lookup_policy(self.object_ref)
         return self.cd.get_policies(policy_paths)
 
-    def create_policy(self, statement: str, policy_type='IAMPolicy', run=True, **kwargs) -> Union[List, None]:
+    def create_policy(self, statement: Dict[str, Any],
```

Comment:
This function is also called by `_set_policy()` in this same file (`clouddirectory.py`), which still passes a statement that is a string. See this bit of code: https://github.com/HumanCellAtlas/fusillade/blob/39ef62ac701763c9f1bdf88173cc478105ba7368/fusillade/clouddirectory.py#L1420-L1435